themes/10up-block-theme/includes/blocks/movie-metadata-director/edit.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+
+/**
+ * @todo Match to FE once we can make dynamic.
+ */
+import { useEntityProp } from '@wordpress/core-data';
+import { __ } from '@wordpress/i18n';
+import { usePost } from '@10up/block-components';
+
+export const BlockEdit = () => {
+	const { postType } = usePost();
+	const [meta] = useEntityProp('postType', 'tenup-movie', 'meta');
+
+	const { tenup_movie_plot = '' } = meta || {};
+
+	let RenderedUI = (
```

Comment:
Instead of reassigning the renderedUI multiple times I would simply use early returns for these cases